### PR TITLE
data: souhaibsfaxi

### PR DESCRIPTION
**Pull request by souhaibsfaxi**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2448"></a>

### CODE REVIEW

File: data/souhaibsfaxi.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "YOUR NAME",
```
Comment (Panquesito7) [inline data/souhaibsfaxi.json:2]: You have to place your name here.

File: data/souhaibsfaxi.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Freelancing",
+      "date": "December 2021",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "Started freelancing",
+      "url": "https://www.eddiejaoude.io/"
```
Comment (Panquesito7) [inline data/souhaibsfaxi.json:26]: I think you should update this link as well.

File: data/souhaibsfaxi.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  "avatar": "https://github.com/souhaibsfaxi.png",
+  "tags": ["javascript", "typescript"],
+  "socials": [
+    
+    { "platform": "github", "url": "https://github.com/souhaibsfaxi" }
+  ],
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/YOUR_GITHUB_USERNAME",
```
Comment (Panquesito7) [inline data/souhaibsfaxi.json:15]: Your GitHub username must be specified here.
```suggestion
      "url": "https://github.com/souhaibsfaxi",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by EmmaDawsonDev** [comment]
How are you doing with the changes requested @souhaibsfaxi ? Let us know if you need any help

File: data/souhaibsfaxi.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Souhaib Sfaxi",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Open Source Enthusiast!",
+  "avatar": "https://github.com/souhaibsfaxi.png",
+  "tags": ["javascript", "typescript"],
+  "socials": [
+    
+    { "platform": "github", "url": "https://github.com/souhaibsfaxi" }
```
Comment (Panquesito7) [inline data/souhaibsfaxi.json:10]: Should be `icon`, AFAIK.
```suggestion
    { "icon": "github", "url": "https://github.com/souhaibsfaxi" }
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: data/souhaibsfaxi.json (hunk 4 above)
Comment (EmmaDawsonDev) [inline data/souhaibsfaxi.json:10]:
```suggestion
    { "icon": "FaGithub", "url": "https://github.com/souhaibsfaxi" }
```

File: data/souhaibsfaxi.json [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+  ],
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/souhaibsfaxi",
+      "icon": "github"
```
Comment (EmmaDawsonDev) [inline data/souhaibsfaxi.json:16]:
```suggestion
      "icon": "FaGithub"
```

File: data/souhaibsfaxi.json [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Freelancing",
+      "date": "December 2021",
+      "icon": "dollar",
```
Comment (EmmaDawsonDev) [inline data/souhaibsfaxi.json:23]:
```suggestion
      "icon": "FaDollarSign",
```

**Review by EmmaDawsonDev — CHANGES_REQUESTED** [review]
You just have a few icon names that need updating

File: data/souhaibsfaxi.json [hunk 7]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Souhaib Sfaxi",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Open Source Enthusiast!",
+  "avatar": "https://github.com/souhaibsfaxi.png",
```
Comment (eddiejaoude) [inline data/souhaibsfaxi.json:6]: This is no longer required
```suggestion

```

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**